graphql/resolvers/__tests/unit.spec.js

```diff
@@ -0,0 +1,11 @@
+const { initTest } = require('../../../helpers/tests');
+const { Query } = require('../unit');
+
+initTest();
+describe('user resolver', () => {
+  it('should get all units', async (done) => {
+    const units = await Query.units();
+    expect(units).toHaveLength(0);
```

Comment:
this does not test if units are coming from the database, length would be zero if it wasn't working too